src/unit_tests/wazuh_db/test_wdb_integrity.c

```diff
@@ -1526,6 +1527,156 @@ void test_wdbi_last_completion_step_fail(void **state)
 
 }
 
+// Test wdb_global_group_hash_cache
+
+void test_wdb_global_group_hash_cache_clear_success(void **state)
+{
+    wdb_global_group_hash_operations_t operation = WDB_GLOBAL_GROUP_HASH_CLEAR;
+    os_sha1 hexdigest;
+    int ret;
+
+    ret = wdb_global_group_hash_cache(operation, hexdigest);
+
+    assert_false(global_group_hash[0]);
+    assert_int_equal(ret, OS_SUCCESS);
+}
+
+void test_wdb_global_group_hash_cache_read_fail(void **state)
```

Comment:
This test isn't independent, it relies on previous execution to work properly. We can set the variables during the test, or modify the setup/teardown.

Reply:
setup/teardown have a very specific name to prepare the wdb_t structure, I preferred to keep it as it is. Since I had to modify the variable on test even using setup/teardown I preferred to modify the variable on every test.

Reply:
I agree !